edx_sga/sga.py

```diff
@@ -239,6 +249,20 @@ def update_staff_debug_context(self, context):
             (name, field.read_from(self))
             for name, field in self.fields.items()]
 
+    def get_module_by_id(self, module_id):
+        """
+        returns student mode object
+        """
+        student_module = None
+        try:
+            student_module = StudentModule.objects.get(pk=module_id)
+        except StudentModule.DoesNotExist:
+            log.info(
```

Comment:
Please don't silence errors for the sake of unit tests. Please create the appropriate data within the unit test instead